### ISSUE TITLE
Update documentation after removing old GUI

### ISSUE THREAD

**Issue by farmaazon** [enso-org/enso#8483]
To be done after #8482 

The documentation should be updated, to no longer keep anything related to the old GUI. Anything useful from old docs (shortcuts?) should be moved to `docs` directory in repo's root.

The PR template should be refreshed to reflect the actual process and remove points which are automated.

**Comment by farmaazon** [comment]
Refinement notes:
* Any missing useful docs (like "how to create visualizations in the new GUI) are not in scope of this task - the assignee should create tasks for them.
